### DISCUSSION TITLE
is the version 3.13.3 release lib  is compiled with efficient implementation of the BLAS and LAPACK library?

### DISCUSSION THREAD

**Discussion by fkshine** [coin-or/Ipopt#456] [Q&A] [answered]
I used the release verison 3.13.3 which is downloaded on git 。but I dont know if it is compiled with  efficient implementation of the BLAS and LAPACK library。anyone can tell me？I am a new user to ipopt, TKS。

**Comment by svigerske** [comment, marked as answer]
https://github.com/coin-or/Ipopt/releases/download/releases%2F3.13.3/Ipopt-3.13.3-win64-msvs2019-md.zip is linked against Intel MKL. So at least as long as you have an Intel CPU, this should be efficient.

There is a README.txt in the zip files that gives more details.

**Comment by cjl88** [comment]
if you are using Linux, you can check out my post https://github.com/coin-or/Ipopt/discussions/453 to verify you are using MKL BLAS.
